server/src/models/load_utils.js

```diff
@@ -42,6 +42,11 @@ const bilingual_remap = (obj, dest_key, source_key) => ({
   [`${dest_key}_fr`]: obj ? obj[`${source_key}_fr`] : null,
 });
 
+// Populate time validation will choke on either the string representation of a negative number or a value of NaN
+// Need to parse strings to numbers ourselves to handle negatives, but take care to preserve nulls which we assume are csv blanks (trying to convert a null gives a NaN)
+// Preserving nulls instead of defaulting to nulls because we still want validation errors to surface when any other data issue produces a NaN!
+const null_preserving_to_number = (value) => _.isNaN( _.toNumber(value) ) ? null : _.toNumber(value);
```

Comment:
This is the only thing holding me up still.

The comment above it is actually wrong. That was my bad, but I suggested it when I was misunderstanding what was going on with the `-` cells in the csvs and thought the error message you'd quoted was about all negative values. You can delete that whole comment.

`null_preserving_to_number` doesn't preserve nulls right now. `_.toNumber(null) === 0`, for a reason that is not completely clear to me (maybe so that `1 + null === 1`, although I'm not certain that's really desirable). Either way. `null_preserving_to_number(null) === 0`, not `null`.

And it looks like you made a commit removing it from use while I was typing it, so that works too. I was going to suggest either dropping it all together or changing it to
```
// toNumber but leaves nulls null, instead of casting them to 0
const null_preserving_to_number = (value) => _.isNull(value) ? null : _.toNumber(value);
```

Either way you'll have to reupdate the snapshots because a bunch of 0's will become nulls. Just skim the snapshots manually to make sure nothing is wrong with them before you commit the updated ones!

Need something like 
```
const null_preserving_to_number = (value) => ( _.isNull(value) || _.isNaN( _.toNumber(value) ) ) ? null : _.toNumber(value)
```

Reply:
I personally think it should just be dropped as it's now only being used in results/populate..!

Reply:
Ah, that sounds good then, go for it!